scripts/radio_manual.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+echo "Radio Frequency Contoller started."
+rigctl -m 214 -r /dev/radio -s 9600
```

Comment:
mamy inne radia w gsach, nazwij proszę plik `radio_fp_manual`, albo coś w ten deseń

Reply:
wrzuć w zmienną środowiskową tak jak jest `PWSAT_GS_NAME`

Reply:
też mi to przyszło na myśl. Port COM też tak zrób może - my mamy tty* hell (bo mamy dwa x prolific... i one są nierozróżnialne).